pyaedt/filtersolutions_core/attributes.py

```diff
@@ -0,0 +1,1528 @@
+
+        Returns
+        -------
+        str
+        """
+        filter_multiple_bands_low_pass_freq_string = self._dll_interface.get_string(
+            self._dll.getMultipleBandsLowPassFrequency
+        )
+        return filter_multiple_bands_low_pass_freq_string
+
+    @filter_multiple_bands_low_pass_frequency.setter
+    def filter_multiple_bands_low_pass_frequency(self, filter_multiple_bands_low_pass_freq_string):
+        self._dll_interface.set_string(
+            self._dll.setMultipleBandsLowPassFrequency,
+            filter_multiple_bands_low_pass_freq_string,
+        )
+
+    @property
+    def filter_multiple_bands_high_pass_frequency(self) -> str:
+        """The multiple bands high pass frequency of combined high pass and band pass filters. The default is `1GHz`.
+
+        Returns
+        -------
+        str
+        """
+        filter_multiple_bands_high_pass_freq_string = self._dll_interface.get_string(
+            self._dll.getMultipleBandsHighPassFrequency
+        )
+        return filter_multiple_bands_high_pass_freq_string
+
+    @filter_multiple_bands_high_pass_frequency.setter
+    def filter_multiple_bands_high_pass_frequency(self, filter_multiple_bands_high_pass_freq_string):
+        self._dll_interface.set_string(
+            self._dll.setMultipleBandsHighPassFrequency,
+            filter_multiple_bands_high_pass_freq_string,
+        )
+
+    @property
+    def order(self) -> int:
+        """Order of filter. The default is `5`.
+
+        Returns
+        -------
+        int
+        """
+        order = c_int()
+        status = self._dll.getOrder(byref(order))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return int(order.value)
+
+    @order.setter
+    def order(self, order: int):
+        status = self._dll.setOrder(order)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def minimum_order_stop_band_attenuation_db(self) -> str:
+        """Filter stop band attenuation in dB for calculation of the filter minimum order.
+        The default is `50`.
+
+        Returns
+        -------
+        str
+        """
+        minimum_order_stop_band_attenuation_db_string = self._dll_interface.get_string(
+            self._dll.getMinimumOrderStopbandAttenuationdB
+        )
+        return minimum_order_stop_band_attenuation_db_string
+
+    @minimum_order_stop_band_attenuation_db.setter
+    def minimum_order_stop_band_attenuation_db(self, minimum_order_stop_band_attenuation_db_string):
+        self._dll_interface.set_string(
+            self._dll.setMinimumOrderStopbandAttenuationdB,
+            minimum_order_stop_band_attenuation_db_string,
+        )
+
+    @property
+    def minimum_order_stop_band_frequency(self) -> str:
+        """Filter stop band frequency for calculation of the filter minimum order.
+        The default is `10 GHz`.
```

Comment:
```suggestion
        The default is ``10 GHz``.
```
Single backtick results in italics type in the doc. It must have double backticks to be rendered as code entity.